api/server/sdk/rest_gateway.go

```diff
@@ -118,6 +119,8 @@ func (s *sdkRestGateway) restServerSetupHandlers() (http.Handler, error) {
 	mux.Handle(prefix,
 		http.StripPrefix(prefix, http.FileServer(swaggerUIBox)))
 
+	mux.Handle("/metrics", promhttp.Handler())
```

Comment:
Maybe change to `/prometheus/metrics` to be clear? Also, is it ok to expose this endpoint without auth? I saw prometheus has an option to hit a metrics endpoint with basic auth, but that might be a pain for configuration.

Reply:
i.e. to distinguish between any other metrics we provide in the future

Reply:
prometheus defaults to /metrics if we move these to another endpoint would require extra configuration